moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -200,7 +199,7 @@ const ProjectView = () => {
   const [dialogState, setDialogState] = useState(null);
   const [anchorElement, setAnchorElement] = useState(null);
   const [snackbarState, setSnackbarState] = useState(DEFAULT_SNACKBAR_STATE);
-  const menuOpen = anchorElement ?? false;
+  const menuOpen = Boolean(anchorElement);
```

Comment:
I encountered an error when anchorElement was being set to an element throwing an error, because it's later passed in as a prop called `open` that is required to be a boolean. This fixes the error, and TBH, I'm confused why this isn't throwing an error when you go into "edit mode" in production.